packages/mds-db/tests/mds-db.spec.ts

```diff
@@ -498,4 +502,38 @@ if (pg_info.database) {
       })
     })
   })
+
+  it('Queries metrics correctly', async () => {
+    const fakeReadOnly = Sinon.fake.returns('boop')
+    Sinon.replace(dbClient, 'makeReadOnlyQuery', fakeReadOnly)
+    const start_time = 42
+    const end_time = 50
+    const provider_id = null
+    const geography_id = null
+    await getAllMetrics({ start_time, end_time, provider_id, geography_id })
+    assert.strictEqual(
+      fakeReadOnly.calledOnceWithExactly(
```

Comment:
should we be doing some additional validation that this query is actually returning expected results or is that not possible without some complicated database pre-seeding logic?

Reply:
Nvm, discussed with max, this would fall under the category of integration tests which is outside of this

Reply:
> Not possible without some complicated database pre-seeding logic.

We'd need to loop the sim in somehow.